helpdesk_mgmt_stage_server_action/models/helpdesk_ticket_stage.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2023 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class HelpdeskTicketStage(models.Model):
+    _inherit = "helpdesk.ticket.stage"
+
+    action_id = fields.Many2one(
+        "ir.actions.server",
+        string="Server Action",
+        domain="[('model_id', '=', 'helpdesk.ticket')]",
+        help="The assigned action will be executed when a ticket is assigned to the stage.",
```

Comment:
Please add which values in passed in context of the action (on model helpdesk.ticket and with ticket ids).